### ISSUE TITLE
Adding the interaction with tokens

### ISSUE THREAD

**Issue by mathboulianne** [eoscanada/eosc#53]
Enabling to get the balance of tokens (e.g. IQ, Karma, Boid, etc) as well as transfer them to another account

**Comment by joshkauffman** [comment]
I thought I left a response to this earlier, but I'm not seeing it now. So apologies if this is duplicated somewhere else. 

This functionality exists through `eosc get balance [account] --contract [string]` but the `eosc get balance [account] --symbol [string]` does not appear to be working. Have tried with multiple symbols, and only EOS worked for me.

**Comment by abourget** [comment]
Is this still an issue?

**Comment by joshkauffman** [comment]
Transferring alts works, but @mathboulianne also mentioned in his issue the ability to get balance of alts. 
would be interesting if we could `eosc get balance [account_name]` and it would print all non-0 balances. I foresee this being handy for a dev who wants to make their own testnet, make some currencies and then play around with them

**Comment by abourget** [comment]
Something like `eosc get popular-balances` ? And we query a bunch of contracts ?

**Comment by joshkauffman** [comment]
Yeah. But again, i don't think it's a priority to have that. A nice-to-have though.
